docs/disable-if-parameter.adoc

```diff
@@ -0,0 +1,83 @@
+:page-title: Disable Parameterized Test Based on Parameter Values
+:page-description: Extends JUnit Jupiter with `@DisableIfParameter`, which selectively disables parameterized tests
+
+The `@DisableIfParameter` annotation can be used to selectively disable parameterized tests based on their parameter values (converted with `toString()`).
+The annotation is only supported on test method level for parameterized tests.
+Unlike the `@Disabled` API provided in JUnit Jupiter, which disables the test on first encounter of the annotation , `@DisableIfParameter` is validated before each parameterized test execution.
+As a consequence, instead of disabling the entire set of parameterized tests, each test can be evaluated and possibly disabled individually.
```

Comment:
```suggestion
As a consequence, instead of disabling the entire set of parameterized tests, each test is evaluated and possibly disabled individually.
```